CONTRIBUTING.md

```diff
@@ -65,11 +76,16 @@ When a pull request is first created, @opendatakit-bot tags it as `needs review`
 
 Small fixes that target very particular bugs may occasionally be merged without a second review.
 
-## Triage Issues [![Open Source Helpers](https://www.codetriage.com/opendatakit/collect/badges/users.svg)](https://www.codetriage.com/opendatakit/collect)
+## Work in progress pull requests
+Work in progress (WIP) pull requests are useful to illustrate a proposed direction and get early feedback before committing to a solution direction. Work in progress pull requests should be clearly identified with [WIP] at the beginning of the title. The description should describe the proposed direction, why it is being considered, what alternatives might be possible and solicit specific feedback from collaborators. 
+
```

Comment:
> When you’ve started a new feature, document, or bug fix, that’s not quite ready for others to review, but you want to let your teammates know that you’re working on the feature. This can be used to prevent the duplication of effort, save work that you’ve started, or complement your team’s workflow.

I found this description in [Six types of PR](https://ben.balter.com/2015/12/08/types-of-pull-requests/), I think it's very good and can be set as a reference.

Besides, should we mention that `WIP PRs will not be merged until you think it's prepared and remove the [WIP] in the title` ?

Reply:
> When you’ve started a new feature, document, or bug fix, that’s not quite ready for others to review, but you want to let your teammates know that you’re working on the feature. This can be used to prevent the duplication of effort, save work that you’ve started, or complement your team’s workflow.

We claim issue to prevent duplication of effort and to let others know that we are working on an issue. So, I feel the main reason of opening a WIP pull request is already added

> WIP PRs will not be merged until you think it's prepared and remove the [WIP] in the title

:+1:

Reply:
Good point, @shobhitagarwal1612. I made some wording changes according to @huangyz0918's suggestions and added the request to remove [WIP].